plugins/stack-overflow-backend/package.json

```diff
@@ -42,5 +42,6 @@
   "files": [
     "dist",
     "config"
-  ]
+  ],
+  "configSchema": "config.d.ts"
```

Comment:
you need to add it to `files` above too, otherwise it won't get shipped in the runtime package (same below)

Reply:
thanks, added her ehttps://github.com/backstage/backstage/pull/12235/commits/a47c1b1407162bdf48906d374eef91a0d4ff2c7b